SilMock/Google/Service/Directory/Resource/Groups.php

```diff
@@ -15,6 +15,17 @@ public function __construct(?string $dbFile = null)
         parent::__construct($dbFile, 'directory', 'groups');
     }
 
+    public function delete(string $groupKey)
```

Comment:
```suggestion
    public function delete(string $groupKey): void
```

Reply:
I would, except that the original Google SDK version (vendor/google/apiclient-services/src/Directory/Resource/Groups.php in MailAdmin) doesn't have the void return type.